website/static/worker.js

```diff
@@ -22,9 +22,8 @@ var prettier = index; // eslint-disable-line
 var parsersLoaded = {};
 
 self.onmessage = function(message) {
-  const options = message.data.options || {
-    parser: "babylon"
-  };
+  var options = message.data.options || {};
```

Comment:
Thanks! The one I have in my autocomplete doens't have it and I need to tweak it manually all the time :P